docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,660 @@
+# Writing Libraries with Cross Platform Tools
+
+By [Phillip Carter](https://github.com/cartermp)
+
+**Some details are subject to change as the toolchain evolves.**
+
+Targeting .NET Core for libraries can be done a number of ways, depending on the platform(s) you wish to support.  Sometimes Visual Studio is not available or a preferred development option.  .NET Core has a set of command line tools you can use instead of Visual Studio.
```

Comment:
I get the point you are wanting to make, but is a bit too much of A vs B. I'd suggest more like this:

> Here is the goal. You can accomplish that goal with the CLI tools. They provide an efficient low-level experience. You can do the same thing in Visual Studio. This document will focus on using the CLI tools directly, however, the goals, concepts and artifacts will all be the same.